### ISSUE TITLE
Using frogbot without GH advanced security?

### ISSUE THREAD

**Issue by tomhmc** [jfrog/frogbot#178]
Is it possible to use frogbot without github advanced security in GH enterprise?

get this error at the moment:

```
  Error: 2 [Error] upload code scanning failed with: POST https://my.github.enterprise/api/v3/repos/org/repo1/code-scanning/sarifs: 403 Advanced Security must be enabled for this repository to use code scanning. []
  Error: The process '/github-actions/actions-runner/_work/_tool/frogbot/[RELEASE]/x64/frogbot' failed with exit code 1
```

seems like it would be useful to have a flag for disabling this functionality:
https://github.com/jfrog/frogbot/commit/2d3b55c2ef9566beb38ad06d25a6dbc0d9eb2dd9#diff-d8b6807ea777814b68ce9cf71f8f78e2afd9dfd682ec923567710b840d43e341R31

(not sure if anything else requires GHAS...?)

**Comment by omerzi** [comment]
Hey @tomhmc,
We were unaware of this situation, and are evaluating a solution for such situations. In any case, Frogbot should work normally under these circumstances. Our team is working on fixing it, and it should be fixed soon. Thank you, I'll keep you posted!

**Comment by omerzi** [comment]
Hey @tomhmc, we have released Frogbot v2.4.0 with the fix. Please let me know if it works for you, thanks!
